samples/storageclass/storageclass.yaml

```diff
@@ -24,6 +24,10 @@ parameters:
   # Storage pool to use on system
   # Optional: false
   storagepool: <STORAGE_POOL> # Insert Storage pool
+  # Protection domain that storage pool above belongs to
+  # Needed if array has two storagepools that share the same name, but belong to different domains
```

Comment:
minor : but belong to different protection domains.

Reply:
fixed